src/routes/OrganizationsList/OrganizationsListTableData.tsx

```diff
@@ -0,0 +1,187 @@
+import {Td} from '@patternfly/react-table';
+import {Skeleton} from '@patternfly/react-core';
+import './css/Organizations.scss';
+import {Link} from 'react-router-dom';
+import {useEffect, useState} from 'react';
+import {fetchOrg} from 'src/resources/OrganizationResource';
+import {
+  fetchRepositoriesForNamespace,
+  IRepository,
+} from 'src/resources/RepositoryResource';
+import {fetchMembersForOrg} from 'src/resources/MembersResource';
+import {fetchRobotsForNamespace} from 'src/resources/RobotsResource';
+import {formatDate} from 'src/libs/utils';
+import ColumnNames from './ColumnNames';
+
+type TeamCountProps = {
+  orgName: string;
+  user: boolean;
+};
+
+type MemberCountProps = {
+  orgName: string;
+  user: boolean;
+};
+
+type RobotCountProps = {
+  orgName: string;
+};
+
+type RepoCountProps = {
+  count: string | number;
+};
+
+type OrgTableProps = {
```

Comment:
this looks the same as `OrganizationsTableItem` can we combine both so we use only one interface everywhere?